docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -266,12 +266,12 @@ The following example of JSON to deserialize contains values that will be deseri
 }
 ```
 
-:::zone-end
-
 Without the custom converter, deserialization puts a `JsonElement` in each property.
 
 The [unit tests folder](https://github.com/dotnet/runtime/blob/81bf79fd9aa75305e55abe2f7e9ef3f60624a3a1/src/libraries/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` namespace has more examples of custom converters that handle deserialization to `object` properties.
 
+:::zone-end
```

Comment:
This removes the link to the tests in the 5.0 pivots. Why do we have it this way?

Reply:
The permalink points to the code as it was for 3.1, I need to make a new link for 5.0 and copy the paragraph with the link.